index.html

```diff
@@ -1 +1,147 @@
+                <li class="list"><a href="#program">Program</a></li>
+                <li class="list"><a href="#about">Join</a></li>
+                <li class="list"><a href="#about">Sponsor</a></li>
+                <li class="list"><a href="#about">News</a></li>
+            </ul>
+        </div>
+        <nav>
+            <div id="hamburger">
+                <i class="fa-sharp fa-solid fa-bars"></i>
+            </div>
+            <div id="nav-container">
+                <div id="top-bar">
+                    <ul class="top-bar-lists">
+                        <li class="top-bar-list"><a href="https://facebook.com"><img class="handle1" src="https://summit.cckorea.org/images/base/facebook.png" alt="facebook page"></a></li>
+                        <li class="top-bar-list"><a href="https://twitter.com"><img class="handle2" src="	https://summit.cckorea.org/images/base/twitter.png" alt="twitter page"></a></li>
+                        <li class="top-bar-list"><a href="#English">English</a></li>
+                        <li class="top-bar-list"><a href="#Mypage">My Page</a></li>
+                        <li class="top-bar-list"><a href="#logout">Logout</a></li>
+                    </ul>
+                </div>
+                <div id="nav-bar">
+                    <div class="nav-bar-container">
+                        <div id="top-logo">
+                            <img src="/assets/logo.png" alt="logo" id="desktop-logo">
+                        </div>
+                        <ul id="nav-lists">
+                            <li class="nav-list"><a href="/about.html">About</a></li>
+                            <li class="nav-list"><a href="#program">Program</a></li>
+                            <li class="nav-list"><a href="#about">Join</a></li>
+                            <li class="nav-list"><a href="#about">Sponsor</a></li>
+                            <li class="nav-list"><a href="#about">News</a></li>
+                            <li class="nav-list campaign"><a class="campaign-txt" href="#campaign">CC Campaign</a></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <section id="headline">
+        <div class="headline-container">
+            <div class="text-container">
+                <h2 class="welcome">"Welcome"</h2>
+                <div class="main-header">
+                    <h1 class="h1_1">Learn It...</h1>
+                    <h1 class="h1_2">Solve It...</h1>
+                </div>
+                <p class="description">An educative learning app for both students and teachers. This app Contains slides and questions on general knowledge as well as engineering</p>
+                <p class="quote">"Failure is an option here. If things are not failing, you are not innovating enough."</p>
+                <p class="elon">(Elon Musk)</p>
+            </div>
+        </div>
```

Comment:
- [ ] As suggested by the previous reviewers, kindly consider adding more text to your main title `Learn It... Solve It...` on the next line to make it similar to the Original design in Behance as it is a project requirement. 
we must remember that in this project, we strive to make our design as similar to the Behance design as possible.
- [ ] Also as suggested by the previous reviewer, Kindly consider recording the loom video once again, in which along with presenting the design of your project, `you also try to show a piece of code in your code editor that you are proud of`. I am writing this requirement here so that it doesn't get missed by you like the previous time. (previous reviewer asked this change in the general code summary)
   - You should also highlight some interesting piece of code or something you built that you are very proud of.